<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LivingRoom</class>
 <widget class="QWidget" name="LivingRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_startliving">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开启直播</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_talkinformation">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>560</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_talking_information">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>聊天信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_talking_information">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>251</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_closeliving">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关闭直播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendtalk_information">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>560</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_camera">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>461</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_allusername">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>直播间用户</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_allusername">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>171</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_roomid">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_video" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>441</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
